packages/react/src/components/DashboardEditor/DashboardEditor.story.jsx

```diff
@@ -591,6 +591,7 @@ export const SummaryDashboardWithInitialValue = () => (
       onCancel={action('onCancel')}
       onSubmit={action('onSubmit')}
       onLayoutChange={action('onLayoutChange')}
+      onEditDataItems={action('onEditDataItems')}
```

Comment:
Any idea why we don't see this action logged in the knobs panel of the story?

Reply:
I dont know the exact reason, But even the other button's actions are not getting displayed properly. (example: cancel, delete button)

Whereas in my local machine I can see that it's triggering the action. (only one difference is it seems to be Class Object whereas submit button is array of object)

![image](https://user-images.githubusercontent.com/18572477/120764262-4ad08a80-c518-11eb-9b88-b29e4ebcdee6.png)